### PR TITLE
chore(JavaScript/ditsmod): upgrade to v3

### PR DESCRIPTION
**Pull request by KostyaTretyak**

### CODE REVIEW

File: javascript/ditsmod/package.json [hunk 1]
```diff
@@ -15,9 +15,9 @@
   "author": "Your Name Here",
   "license": "MIT",
   "dependencies": {
-    "@ditsmod/body-parser": "~2.17.1",
-    "@ditsmod/core": "~2.61.1",
-    "@ditsmod/routing": "~2.7.0"
+    "@ditsmod/body-parser": "^3.0.0-alpha.2",
+    "@ditsmod/core": "^3.0.0-alpha.2",
```
Comment (waghanza) [inline javascript/ditsmod/package.json:19]: When do you plan to release a non beta / alpha ?

Comment (KostyaTretyak) [reply javascript/ditsmod/package.json:19]: No plan, for now =).

Comment (waghanza) [reply javascript/ditsmod/package.json:19]: Not sure it's a good idea to use beta/alpha, but policy is not clear enough for that.

Will accept this for now

**Review by waghanza — APPROVED** [review]

File: javascript/ditsmod/package.json (hunk 1 above)
Comment (KostyaTretyak) [reply javascript/ditsmod/package.json:19]: Thank you. Do not doubt that when a stable version is released, I will definitely update it on your project as well.

**Pull request merged**